cartridges/int_adyen_SFRA/cartridge/controllers/Adyen.js

```diff
@@ -195,7 +292,7 @@ function isMethodTypeBlocked(methodType) {
 /**
  * Get OriginKey for Secured Fields
  */
-server.get('GetConfigSecuredFields', server.middleware.https, function (req, res, next) {
+server.get('getConfigurationComponents', server.middleware.https, function (req, res, next) {
```

Comment:
```suggestion
server.get('GetConfigurationComponents', server.middleware.https, function (req, res, next) {
```
Not sure if PascalCase is a requirement for Salesforce,  up to you if you want to change it for consistency

Reply:
Done